addons/mass_mailing/views/mailing_list_views.xml

```diff
@@ -225,7 +225,7 @@
                                         <field name="contact_count_email"/>
                                     </span>
                                     <br/>
-                                    <span class="text-secondary">
```

Comment:
Could you squash all "text-secondary -> text-muted" changes with the "Milk colors" commit?

Reply:
@stefanorigano Done!